core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -392,12 +413,15 @@ private[spark] class SparkRuntimeException(
 private[spark] class SparkSecurityException(
     errorClass: String,
     errorSubClass: Option[String] = None,
-    messageParameters: Array[String])
+    messageParameters: Map[String, String])
```

Comment:
This is tested by QueryExecutionErrorsSuite "RESET_PERMISSION_TO_ORIGINAL: can't set permission"